lib/stsci/tools/asnutil.py

```diff
@@ -14,7 +14,9 @@
 from astropy.io import fits
 import numpy as N
 import os.path, time
+from distutils.version import LooseVersion
 
+ASTROPY_VER_GE13 = LooseVersion(astropy.__version__) >= LooseVersion('1.3')
```

Comment:
I  am wondering about the way to compare versions here. For example, if I have an earlier development version of astropy

```
LooseVersion('1.3.dev1555') > LooseVersion('1.3')
True # wrong 

from astropy.utils import minversion
minversion(astropy,'1.3.dev1555') >minversion(astropy,'1.3')
False
```